### PR TITLE
Admin Gen Future: Use block name/import as defined in field or column config

### PR DESCRIPTION
**Pull request by nsams**
Previously the configured block was ignored and a block found but our block detection heuristics was used.

We keep the heuristics for copy/paste in grid.

### CODE REVIEW

File: demo/admin/src/products/future/generated/ProductForm.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ import {
 } from "./ProductForm.gql.generated";
 
 const rootBlocks = {
-    image: DamImageBlock,
+    image: PixelImageBlock,
```
Comment (nsams) [inline demo/admin/src/products/future/generated/ProductForm.tsx:53]: this change is wrong, but that is beacuse it's defined wrong in ProductForm.cometGen.ts

Comment (nsams) [reply demo/admin/src/products/future/generated/ProductForm.tsx]: I fixed it in this PR (required to make it build)

File: demo/admin/src/products/future/generated/ProductForm.tsx [hunk 2]
```diff
@@ -255,7 +255,7 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
                             label={<FormattedMessage id="product.availableSince" defaultMessage="Available Since" />}
                         />
                         <Field name="image" isEqual={isEqual}>
-                            {createFinalFormBlock(PixelImageBlock)}
+                            {createFinalFormBlock(DamImageBlock)}
```
Comment (Ben-Ho) [inline demo/admin/src/products/future/generated/ProductForm.tsx:258]: @nsams should we close or merge #1904?

**Review by Ben-Ho — DISMISSED** [review]

File: demo/admin/src/products/future/generated/ProductForm.tsx (hunk 2 above)
Comment (nsams) [reply demo/admin/src/products/future/generated/ProductForm.tsx:258]: close? we need both, those are two different issues

**Review by johnnyomair — APPROVED** [review]

**Comment by johnnyomair** [comment]
@nsams conflicts 🙄

**Review by Ben-Ho — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**